src/csv.hpp

```diff
@@ -0,0 +1,1248 @@
+// Copyright: (2012-2015) Ben Strasser <code@ben-strasser.net>
```

Comment:
Is

https://github.com/Project-OSRM/osrm-backend/blob/ffd6311e7ddabcf9b6c829e7c79a6e708d66dc51/include/updater/csv_file_parser.hpp

"good enough"?